docs/pages/router/basics/layout.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Navigation layouts in Expo Router
+description: Learn how to construct different relationships between pages by using directories and layout files.
+sidebar_title: Layout
+---
+
+import { FileTree } from '~/ui/components/FileTree';
+
+Each directory within the **app** directory (including **app** itself) can define a layout in the form of a **\_layout.tsx** file inside that directory. This file defines how all the pages within that directory are arranged. This is where you would define a stack navigator, tab navigator, drawer navigator, or any other layout that you want to use for the pages in that directory. The layout file exports a default component that is rendered prior to whatever page you are navigating to within that directory.
+
+Let's look at a few common layout scenarios.
+
+## Root layout
+
+Virtually every app will have a **\_layout.tsx** file directly inside the **app** directory. This is the root layout and represents the entry point for your navigation. In addition to describing the top-level navigator for your app, this file is where you would put initialization code that may have previously gone inside an **App.jsx** file, such as loading fonts, interacting with the splash screen, or adding context providers.
+
+Here's an example root layout:
+
+```tsx app/_layout.tsx
+import { useFonts } from 'expo-font';
+import { Stack } from 'expo-router';
+import * as SplashScreen from 'expo-splash-screen';
+import { useEffect } from 'react';
+
+SplashScreen.preventAutoHideAsync();
+
+export default function RootLayout() {
+  const [loaded] = useFonts({
+    SpaceMono: require('../assets/fonts/SpaceMono-Regular.ttf'),
+  });
+
+  useEffect(() => {
+    if (loaded) {
+      SplashScreen.hideAsync();
+    }
+  }, [loaded]);
+
+  if (!loaded) {
+    return null;
+  }
+
+  return <Stack />;
+}
+```
+
+The above example shows the splash screen initially and then renders a stack navigator once the fonts are loaded, which will cause your app to proceed to the initial route.
+
+## Stacks
+
+You can implement a stack navigator in your root layout, as shown above, or in any other layout file inside a directory. Let's suppose you have a file structure with a stack inside of a directory:
+
+<FileTree
+  files={[
+    ['app/products/\_layout.tsx'],
+    ['app/products/index.tsx'],
+    ['app/products/[productId].tsx'],
+    ['app/products/accessories/index.tsx'],
+  ]}
+/>
+
+If you want everything inside of the **apps/products** directory to be arranged in a stack relationship, inside the **\_layout.tsx** file, return a `Stack` component:
+
+```tsx _layout.tsx
```

Comment:
````suggestion
```tsx app/products/_layout.tsx
````

There files name in these code snippets are a bit inconsistent. Sometimes, we're referring to the file name and sometimes to the file path. Let's add the complete path. This might help the new developers to scan/not get confused as to which layout file is being referred to in a code example.